elements/k-new-post.html

```diff
@@ -36,6 +37,7 @@
       <div>
         <k-title title="{{titleValue}}"></k-title>
         <k-short-content short-content="{{shortContentValue}}"></k-short-content>
+        <k-header-image uploadUrl="[[imageHost]]"></k-header-image>
```

Comment:
add the output here, so you can access it from outside

e.g

```
<k-header-image
  uploadUrl="[[imageHostUrl]]"
  header-image="{{headerImageValue}}"
></k-header-image>
```

Then from outside, you can use the `headerImageValue` variable, gud luck :dancer: